src/dlgTriggerEditor.cpp

```diff
@@ -345,7 +345,7 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     connect( showSearchAreaAction, SIGNAL(triggered()), this, SLOT( slot_show_search_area()));
 
     QAction * saveAction = new QAction( QIcon( QStringLiteral( ":/icons/document-save-as.png" ) ), tr("Save Item"), this);
-    //saveAction->setShortcut(tr("Ctrl+S"));
+    saveAction->setShortcut(tr("Ctrl+S"));
```

Comment:
After reviewing the Qt Documentation for the `QKeySequence` class under the heading __Keyboard Layout Issues__:
> Although some developers might resort to fully specifying all the modifiers they use on their keyboards to activate a shortcut, this will also result in unexpected behavior for users of different keyboard layouts.
For example, a developer using a British keyboard may decide to specify "`Ctrl+Shift+=`" as the key sequence in order to create a shortcut that coincidentally behaves in the same way as `Ctrl plus`. However, the `=` key needs to be accessed using the `Shift` key on Norwegian keyboard, making the required shortcut effectively `Ctrl Shift Shift =` (an impossible key combination).
>
> As a result, both human-readable strings and hard-coded key codes can both be problematic to use when specifying a key sequence that can be used on a variety of different keyboard layouts. Only the use of standard shortcuts guarantees that the user will be able to use the shortcuts that the developer intended.
>
> Despite this, we can address this issue by ensuring that human-readable strings are used, making it possible for translations of key sequences to be made for users of different languages. This approach will be successful for users whose keyboards have the most typical layout for the language they are using. 

I was going to suggest we use `saveAction->setShortcut( QKeySequence::Save )` instead.  It will get mapped to `Ctrl+S` currently for the __en-US__ case (and other languages in the future where that is appropriate) and to national other values as needed.

However, this might make the use further down of `Ctrl_Shift+S` problematic, as THAT shortcut normally means a "SaveAs" action but it is being used as "SaveToDisk" whereas there is NOT currently a shortcut for "SaveAsToDisk"...

How about:
`saveAction->setShortcut(tr("Alt+S"))` for this one
and:
`profileSaveAction->setShortcut( QKeySequence::Save )` i.e. Ctrl-S by default
`saveProfileAsAction->setShortcut( QKeySequence::SaveAs )` i.e. Ctrl-Shift-S by default
for the other ones further down?

_Also it might be worth noting that Heiko previously commented out the Ctrl-S binding in https://github.com/Mudlet/Mudlet/commit/96bb8d2ed38df344403342439f2aa45d906748cc on 2009-03-14 17:40:17 with the comment "Fix: couldnt bind control+s keys" after re-purposing it from the `showSearchAreaAction` in https://github.com/Mudlet/Mudlet/commit/22e0c3ba32b2cb68949c0a03417c92bca228b7d9 2009-03-07 13:43:20 - it might be worthwhile establishing what broke back then to see if it is still broken...!_